jaworski_cellseg/jwski_widget.py

```diff
@@ -19,47 +21,88 @@ def __init__(self, napari_viewer: napari.viewer.Viewer, parent=None):
         super().__init__(parent)
         self.viewer = napari_viewer
         self.physical_sizes = {"x": 0.0, "y": 0.0, "z": 0.0}
+        # Read the configuration file
+        self.config_file_path = Path(__file__).parent / "config"  / "config.yaml"
+        with open(self.config_file_path, "r") as f:
+            self.config = yaml.safe_load(f)
+            self.configurations = list(self.config.get('settings', {}).keys())
+            self.current_config = self.config['settings'].get('default', {})
```

Comment:
Would getting the settings group from the file once and reuse it work here?
```suggestion
            settings = self.config.get('settings', {})
            self.configurations = list(settings.keys())
            self.current_config = settings.get('default', {})
```

Reply:
Done